### PR TITLE
add preselected `select` and `multiselect` values to `test valid open_full_form_modal`

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR adds fields for a preselected select and multiselect form for the command `test valid open_full_form_modal`

The PR depends on https://github.com/mattermost/mattermost-mobile/pull/5640

#### Ticket Link
N/A

### CODE REVIEW

**Review by mickmister — APPROVED** [review]
LGTM

**Review by larkox — APPROVED** [review]
LGTM. Is this feature already working for users/channels? If so, it would be great also to add commands/modals for those here. If not, we can defer until we actually support that.

File: routers/mattermost/fForm.go [hunk 1]
```diff
@@ -150,6 +150,25 @@ func fFullFormOK(w http.ResponseWriter, r *http.Request, c *apps.CallRequest) {
 					Type:  apps.FieldTypeUser,
 					Label: "user",
 				},
+				{
+					Name:       "staticpreselected",
+					Type:       apps.FieldTypeStaticSelect,
+					ModalLabel: "static - preselected",
```
[the remark this answers is not in the document]
Comment (jfrerich) [reply routers/mattermost/fForm.go:156]: What am I missing here?  When I test this, I open the modal with `/test valid open_full_form_modal` and there are no other options?

**Comment by levb** [comment]
@mickmister this PR should either be re-done for the new test app, or closed altogether?

**Comment by mickmister** [comment]
@levb This should be re-done, though the project structure may be so different that we should close and open a new PR

**Pull request closed without merging**